news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+
+<p align="center"><iframe src="https://player.twitch.tv/?video=v167002207&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe></p>
+
+The [250 - 500 rank bracket](http://challonge.com/znsjvw11) saw well-known all-around player [-GN](https://osu.ppy.sh/u/895581) pitted against [smh](https://osu.ppy.sh/u/1629553), a former tourney winner himself in the 2016 osu! Indonesia Tournament. In a surprising result in the Winners Final, smh had knocked -GN down to the Losers Bracket in a back-and-forth clash where he would ultimately triumph in a tiebreaker, and as such -GN entered the Grand Final seeking revenge. The match would prove to be a strategic one as early as the ban stage: recognizing smh to be a skilled Double Time player, -GN banned Approach Rate 10.3 DT pick [Utagumi Setsugekka - Yumemi Sunrise](https://osu.ppy.sh/b/338862), while smh in turn targeted -GN’s unparalleled reading ability by banning [Eisyo-kobu - Oriental Blossom](https://osu.ppy.sh/b/1242790). As the winner of the roll, -GN selected the first map, but despite having his choice he was defeated on [xi - Aragami](https://osu.ppy.sh/b/547874). The players would split the next two picks before smh managed to hold his own on technically challenging [Camellia vs Akira Complex - Reality Distortion](https://osu.ppy.sh/b/1171267) to take the lead at 3-1. Both players won their next two picks each, leaving it match point for smh at 5-3. -GN selected Hidden map [ginkiha - Anemoi](https://osu.ppy.sh/b/1316661), and it would be here that smh seized his opportunity as -GN broke partway through the map while smh held his combo to an insurmountable lead to seal a 6-3 victory.
+
+[Full match information](https://osu.ppy.sh/community/matches/35241598)
+
+<p align="center"><iframe src="https://player.twitch.tv/?video=v167002950&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe></p>
+In the [500 - 1000 bracket](http://challonge.com/h015rw5f), [Exarch](https://osu.ppy.sh/u/5370153) advanced through the Winners Bracket in dominating fashion, dropping only 4 maps en route to the Grand Final. There he would meet [pyccknn](https://osu.ppy.sh/u/3618773) in a repeat of their Winners Final matchup after pyccknn triumphed convincingly in the Losers Bracket Final. Unfortunately, real-world circumstances caught up to pyccknn, as he was unable to utilize his normal tablet and PC setup to play. After Exarch took an early 3-0 lead playing his standard tablet and keyboard style despite pyccknn being forced into mouse-only, he recognized the situation and they played out the rest of the series with both only using their mouse. In the exciting and hilarious showmatch that followed, Exarch ended up winning 6-5 on the tiebreaker, [Demetori - Youkai no Yama ~ Mysterious Mountain](https://osu.ppy.sh/b/1260538). 
+
+[Full match information](https://osu.ppy.sh/community/matches/35247144)
```

Comment:
same as above re: full match information, wont mention this again but you get the idea